src/scenes/Login/Login.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react'
+import { useHistory } from 'react-router-dom'
+import './_login.scss'
+import PropTypes from 'prop-types'
+
+const Login = ({ setFoundUser }) => {
+  const history = useHistory()
```

Comment:
Do consider when importing more functions. This should work
![Screen Shot 2021-06-26 at 5 44 44 PM](https://user-images.githubusercontent.com/14868839/123510511-40805700-d6a6-11eb-8f99-8783bda75398.png)

Reply:
Do u know why?

Reply:
Now, i don't need setFoundUser, i removed it, and edited history with your idea!
![image](https://user-images.githubusercontent.com/62316127/123610213-027a5480-d7c6-11eb-8dac-795cf81cc7f1.png)